pkg/api-server/definitions/api.go

```diff
@@ -1,17 +1,49 @@
 package definitions
 
 import (
+	"github.com/kumahq/kuma/pkg/core/resources/apis/system"
+	"github.com/kumahq/kuma/pkg/core/resources/model"
 	core_model "github.com/kumahq/kuma/pkg/core/resources/model"
 	core_rest "github.com/kumahq/kuma/pkg/core/resources/model/rest"
+	"github.com/kumahq/kuma/pkg/core/resources/registry"
 )
 
-func AllApis() core_rest.Api {
-	return Apis(All...)
+type ResourceWsDefinition struct {
+	Type     model.ResourceType
+	Path     string
+	ReadOnly bool
+	Admin    bool
```

Comment:
nit: maybe in a second PR, rename this to `AdminOnly` so it matches a bit better